src/utils/daos.c

```diff
@@ -909,9 +910,13 @@ main(int argc, char *argv[])
 	command_hdlr_t		hdlr = NULL;
 	struct cmd_args_s	dargs = {0};
 
-	/* argv[1] is RESOURCE or "help";
+	/* argv[1] is RESOURCE or "help" or "version";
 	 * argv[2] if provided is a resource-specific command
 	 */
+	if (argc <= 2 || strcmp(argv[1], "version") == 0) {
```

Comment:
This will now show the version if no other argument is supplied, whereas before it showed the help output. Was that intentional? The control plane utilities will only display the version if the first argument is "version".

Reply:
agree. It seems we want
daos (no arguments) - invoke help_hdlr
daos help - same
daos version - print version and return/exit

Reply:
Oops, you are right guys, thx.